client-v2/src/services/editionsApi.ts

```diff
@@ -0,0 +1,57 @@
+import { EditionIssue } from 'types/Edition';
+import { Moment } from 'moment';
+import pandaFetch from './pandaFetch';
+
+const dateFormat = 'YYYY-MM-DD';
+
+export const fetchIssuesForDateRange = async (
```

Comment:
This endpoint while useful is not going to be part of the editions first pass.

Reply:
Yes indeed, I refer to that in the writeup. We decided to leave in here but can remove if you feel strongly